examples/autoscaler-webhook/Makefile

```diff
@@ -39,16 +45,20 @@ root_path = $(realpath $(project_path)/../..)
 
 # Build a docker image for the server, and tag it
 build:
-	cd $(root_path) && docker build -f $(project_path)/Dockerfile --tag=$(autoscaler_webhook_tag) .
+	cd $(project_path) && docker build -f $(project_path)/Dockerfile --tag=$(autoscaler_webhook_tag) .
```

Comment:
Can you please share if there is a better alternative to changing the $(root_path)? I attempted to use $(root_path) as a substitution in the build script, but it didn't work as expected.

Reply:
Not entirely sure why the change in name here (although project vs root I don't really mind).

That being said, I think you could revert this lines change, and with my suggestions below this will work.

Reply:
yes, root_path works!